akka-docs-dev/rst/java/http/routing-dsl/request-vals/oauth2-authenticator.rst

```diff
@@ -0,0 +1,47 @@
+.. _oauth2-authenticator-java:
+
+Request Values: OAuth 2 Bearer Token Authentication
+===================================================
+
+An abstract class to implement Oauth 2 bearer token authentication
+
+Description
+-----------
+Allows to protect one of more routes with authentication in the form of a OAuth2 Bearer Token. For more information
+about OAuth 2 Bearer Token see `RFC6750`_.
+
+.. _RFC6750: https://tools.ietf.org/html/rfc6750
+
+To use it you subclass ``OAutht2Authenticator`` and implement the ``authenticate`` method
```

Comment:
hint, we sometimes use 

```
:class:`OAutht2Authenticator`
```

(currently does nothing, but we're hoping to get tooling around it eventually)